VKApp/VKApp/Views/GroupUserTableViewCell.swift

```diff
@@ -5,35 +5,21 @@ import UIKit
 
 /// Ячейка с группой и фотографией группы в которых пользователь не состоит
 final class GroupUserTableViewCell: UITableViewCell {
-    // MARK: - Constants
-
-    private enum Constants {
-        static let emptyText = ""
-        static let transformScaleText = "transform.scale"
-    }
-
     // MARK: - Private Outlets
 
     @IBOutlet private var groupNameLabel: UILabel!
     @IBOutlet private var groupPhotoImageView: UIImageView!
 
     // MARK: - Public Methods
 
-    func configure(group: VKGroups, photoService: PhotoService?, indexPath: IndexPath) {
+    func configure(group: VKGroups, photoService: PhotoService?, row: Int) {
         selectionStyle = .none
         groupNameLabel.text = group.name
+        let indexPath = IndexPath(row: row, section: Int.zero)
```

Comment:
```suggestion
        let indexPath = IndexPath(row: row, section: Int.zero)
```
IndexPath зачем тебе такое если можно просто индекс отдавать?